tests/05_ipsets/02_usage

```diff
@@ -146,6 +146,10 @@ table inet fw4 {
 	#
 
 
+	#
+	# Limits
```

Comment:
Do you have a coverage metric to fill by these nop tssts?

Reply:
I was adding that header unconditionally but reviewing the code, fw4 avoids adding headers when not needed. I just pushed a fix adding Limits section only when needed.